jenkins_jobs/modules/hipchat_notif.py

```diff
@@ -139,20 +135,18 @@ def gen_xml(self, parser, xml_parent, data):
         XML.SubElement(pdefhip, 'startNotification').text = str(
             hipchat.get('notify-start', hipchat.get('start-notify',
                                                     False))).lower()
-
-        if version >= pkg_resources.parse_version("0.1.5"):
-            XML.SubElement(pdefhip, 'notifySuccess').text = str(
-                hipchat.get('notify-success', False)).lower()
-            XML.SubElement(pdefhip, 'notifyAborted').text = str(
-                hipchat.get('notify-aborted', False)).lower()
-            XML.SubElement(pdefhip, 'notifyNotBuilt').text = str(
-                hipchat.get('notify-not-built', False)).lower()
-            XML.SubElement(pdefhip, 'notifyUnstable').text = str(
-                hipchat.get('notify-unstable', False)).lower()
-            XML.SubElement(pdefhip, 'notifyFailure').text = str(
-                hipchat.get('notify-failure', False)).lower()
-            XML.SubElement(pdefhip, 'notifyBackToNormal').text = str(
-                hipchat.get('notify-back-to-normal', False)).lower()
+        XML.SubElement(pdefhip, 'notifySuccess').text = str(
+            hipchat.get('notify-success', False)).lower()
+        XML.SubElement(pdefhip, 'notifyAborted').text = str(
+            hipchat.get('notify-aborted', False)).lower()
+        XML.SubElement(pdefhip, 'notifyNotBuilt').text = str(
+            hipchat.get('notify-not-built', False)).lower()
+        XML.SubElement(pdefhip, 'notifyUnstable').text = str(
+            hipchat.get('notify-unstable', False)).lower()
+        XML.SubElement(pdefhip, 'notifyFailure').text = str(
+            hipchat.get('notify-failure', False)).lower()
+        XML.SubElement(pdefhip, 'notifyBackToNormal').text = str(
```

Comment:
This is in fact not something our incarnation plugin cans, but I've tested that it does no harm if it's there.